SU2_CFD/include/solvers/CFEASolver.hpp

```diff
@@ -55,8 +55,8 @@ class CFEASolver : public CSolver {
   su2double Conv_Check[3];          /*!< \brief Current values for convergence check: UTOL, RTOL, ETOL. */
   su2double FSI_Conv[2];            /*!< \brief Values to check the convergence of the FSI problem. */
 
-  unsigned long idxIncrement;       /*!< \brief Index of the current load increment */
-  su2double loadIncrement;          /*!< \brief Coefficient that determines the amount of load which is applied. */
+  unsigned long idxIncrement = 0;   /*!< \brief Index of the current load increment */
```

Comment:
just a matter of taste, but I like braced initialization for these, e.g., 

```unsigned long idxIncrement{0};```

Reply:
I like `=` :)